api/run.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
 
-make build
+if [ -z "$MODEL" ]
+then
+    echo "Please set the MODEL_FILE environment variable"
+    exit 1
+fi
+
+if [ -z "$MODEL_DOWNLOAD_URL" ]
+then
+    echo "Please set the MODEL_DOWNLOAD_URL environment variable"
+    exit 1
+fi
+
+
+if [ ! -f $MODEL ]; then
+    echo "Model file not found. Downloading..."
+    curl -L -o $MODEL $MODEL_DOWNLOAD_URL
```

Comment:
```suggestion
    apt-get update --yes --quiet
    apt-get install --yes --quiet curl
    curl -L -o $MODEL $MODEL_DOWNLOAD_URL
```

We can install `curl` here if model doesn't exist.